submissions/oluwatosinOke/phpAndDatabases/customerOrder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+require_once("connection.php");
+try {
+    $sql=   "SELECT  other_items.unit_price, other_items.quantity, other_items.total_amount, 
+                orders.ordercreated_at, orders.id, products.product_name, customers_address.street_address,
+                customers_address.city, customers_address.state, customers_address.country
+            FROM other_items
+            LEFT JOIN orders ON orders.id = other_items.order_id
+            LEFT JOIN products ON products.id = other_items.product_id
+            LEFT JOIN customers_address ON orders.customer_orders = customers_address.customer_id";
+    $result= $pdo->query($sql);
+    if ($result->rowCount()>0){
```

Comment:
the idea of this task is to store the result returned from this SQL query in a cookie, Each time I reload this page, I don't want to always run this SQL query but instead, I want to get it from the data stored in the cookie as long as the cookie has not expired. If the cookie is has expired then run the SQL query
With your current implementation, each time I reload this page, I am always executing this SQL query.